fmriprep/workflows/base.py

```diff
@@ -623,6 +623,10 @@ def init_single_subject_wf(subject_id: str):
         num_bold=len(bold_runs)
     )
 
+    # Before initializing BOLD workflow, select/verify anatomical target for coregistration
+    if config.workflow.bold2anat_init in ('auto', 't2w'):
+        config.workflow.bold2anat_init = 't2w' if subject_data['t2w'] else 't1w'
```

Comment:
#3202 allowed us to proceed with derivatives even when the raw anatomicals are missing. Might as well keep it up.

```suggestion
        config.workflow.bold2anat_init = 't2w' if subject_data['t2w'] or 't2w_preproc' in anatomical_cache else 't1w'
```